### PR TITLE
Add new "prize list" for Mining Vendors: MODsuit Modules.

### PR DESCRIPTION
**Pull request by Venuska1117**
I don't know if prices for modules are too high or too low, and idk if i should request TM or not..

## What Does This PR Do
Add new "Prize list" to Mining Vendor, this new pool is expansion for MODsuit modules, now miners can buy "basic" modules for their mining modsuit.

## Why It's Good For The Game
Now miners can get their MODsuit modules during low-pop or if robotics refuse to do their jobs, of course you still require to go and get better upgrades like extended storage or thermal cooling thing, so it still require interaction with robotics for upgrades.

## Images of changes
image

## Testing
i will check in moment.

## Changelog
:cl: Venuska1117
add: Now you can buy "basic" MODsuit modules in mining vendor.
tweak: Changed mining MODsuit and Advanced Jetpack position to new category.
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
Is that the advanced jetpack or the normal one?

**Comment by CinnamonSnowball** [comment]
> if robotics refuse to do their jobs

This is an IC issue that frankly should be resolved IC, not via code.

Already voiced my personal opinion on the Discord but figured it might be good to have here: not a fan of making Mining even more independent from the station than they already are. We should be _encouraging_ interdepartmental cooperation, not providing ways to avoid it - miners already spend the vast majority of their shift on Lavaland.

**Comment by Venuska1117** [comment]
> > if robotics refuse to do their jobs
> 
> This is an IC issue that frankly should be resolved IC, not via code.
> 
> Already voiced my personal opinion on the Discord but figured it might be good to have here: not a fan of making Mining even more independent from the station than they already are. We should be _encouraging_ interdepartmental cooperation, not providing ways to avoid it - miners already spend the vast majority of their shift on Lavaland.

yes but it only contain _basic_ modules, if you want more advanced ones, you still need go to Robotics.

**Comment by Banderblitz** [comment]
As someone who frequently mines myself, I'd say no to the drill despite how convenient it would be. It would be like putting the advanced plasma cutter inside the vendor and the drill requires an actually decent amount of tech/materials. It being available from vendors would simply mean it's never printed from science.

Otherwise, most of what you're adding is actually never really even used on mining modsuits, so it's rather pointless to add, it will just bloat the vendor. One could argue the flashlight or mesons, but miners have a plethora of other light sources and miners are naturally given mesons as well as can buy a form of them already. Also as said earlier part of it too is encouraging miners to actually interact with the station more than drop ore, leave.

TLDR: Personally, I'd say no to the modules, would just discourage miners from interacting with station and the drill is actually a good piece of tech that should'nt be put inside the vendor.

**Comment by Venuska1117** [comment]
I want request TM on this to see how it would go on live server first.

**Comment by Denthamos** [comment]
Will say right now this is not the path forward we as a server want to take at all in regards to **any** job/department becoming more self sufficient and not working together and communicating with other departments.  Anything similar like this which would keep players from going to another department/job for help, items, or stuff that would further enhance their gameplay will likewise be not liked well at all.

**Comment by Venuska1117** [comment]
Closing this PR, this pr was "test" seeing what staff would think about it, considering its mostly negative and MOD suits are still new, this _maybe_ will be reopened in future when mining would change and this sort of stuff will be acceptable.
